src/tests/LitePsmJob-integration.t.sol

```diff
@@ -0,0 +1,166 @@
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+pragma solidity 0.8.13;
+
+import "./DssCronBase.t.sol";
+
+import {LitePsmJob} from "../LitePsmJob.sol";
+import {LitePsmLike} from "../LitePsmJob.sol";
+
+interface DaiAbstract {
+    function approve(address, uint256) external;
+    function balanceOf(address) external view returns (uint256);
+}
+
+interface GemLike {
+    function approve(address, uint256) external;
+    function balanceOf(address) external view returns (uint256);
+    function decimals() external view returns (uint8);
+    function transfer(address, uint256) external;
+    function transferFrom(address, address, uint256) external;
+}
+
+interface VatLike {
+    function frob(bytes32, address, address, address, int256, int256) external;
+    function hope(address) external;
+    function slip(bytes32, address, int256) external;
+    function ilks(bytes32) external view returns (uint256, uint256, uint256, uint256, uint256);
+    function debt() external view returns (uint256);
+    function Line() external view returns (uint256);
+    function urns(bytes32, address) external view returns (uint256, uint256);
+    function file(bytes32 ilk, bytes32 what, uint data) external;
+    function file(bytes32 what, uint data) external;
+}
+
+contract LitePsmJobIntegrationTest is DssCronBaseTest {
+
+    using GodMode for *;
+
+    uint256 constant MILLION_WAD = MILLION * WAD;
+
+    LitePsmLike public litePsm;
+    LitePsmJob public litePsmJob;
+    address public gem;
+    address public dai;
+    address public pocket;
+    address vat;
+    bytes32 ilk;
+
+    // --- Events ---
+    event Chug(uint256 wad);
+    event Fill(uint256 wad);
+    event Trim(uint256 wad);
+    event Work(bytes32 indexed network);
+
+    function setUpSub() virtual override internal {
+        litePsm = LitePsmLike(dss.chainlog.getAddress("MCD_LITE_PSM_USDC_A"));
+        pocket = dss.chainlog.getAddress("MCD_LITE_PSM_POCKET_USDC_A");
+        dai = dss.chainlog.getAddress("MCD_DAI");
+        litePsmJob = new LitePsmJob(address(sequencer), litePsm, MILLION_WAD, MILLION_WAD, MILLION_WAD);
+        gem = litePsm.gem();
+        ilk = litePsm.ilk();
+        vat = litePsm.vat();
+        // give auth access to this contract (caller) to vat for manipulating params
+        GodMode.setWard(vat, address(this), 1);
+    }
+
+    function test_fill() public {
+        // GodMode.setWard(address(litePsm), address(this), 1);
```

Comment:
You most likely want to remove this line, since it's not used.
If you ever needed this, I believe the best place to put would be inside `setUpSub`.
```suggestion
```

Reply:
removed in [092a8cd](https://github.com/makerdao/dss-cron/pull/25/commits/092a8cdcd7eb3470b95287314b863123169eb59e)